ds3-autogen-parser/src/main/resources/typeNameMap.json

```diff
@@ -74,7 +74,7 @@
     },
     {
       "ContractName": "NodeApiBean",
-      "SdkName": "Ds3Node"
+      "SdkName": "Endpoint"
```

Comment:
they are sort of overlapping, not quite a superset in the way that bucket vs bucketDetails was... but I'm def open to changing it.. I'll ponder this a bit more